src/wdl/tasks/terra_tasks.wdl

```diff
@@ -247,4 +249,35 @@ task addReadsField {
     output {
         String reads_json = read_string("reads.json")
     }
+}
+
+task ValidateFileStatus {
```

Comment:
The `ValidateFileStatus` task seems to be very specific to GDC - should this be moved to `ValidateFileStatus.wdl` instead?

Reply:
So this task is actually used in the main wdl. So I moved it here so it wouldnt have to be duplicated in both

Reply:
Makes sense, missed that thanks!